README.md

```diff
@@ -44,6 +44,9 @@ Run your [Spark on AWS EMR](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/e
 
   //(optional) Total number of instances, including master node. The default value is 1.
   sparkInstanceCount := 2
+
+   //(optional) 
+    sparkSubmitConfs := "--conf spark.executor.cores=10 --conf spark.executor.instances=2 ..."
```

Comment:
This should be a `Map[String, String]`?

Reply:
Fixed . Sorry about the README mess I don't what was I thinking.